com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/common/SessionControllerTest.java

```diff
@@ -59,6 +59,8 @@
 	private CommandCodec commandCodec;
 	@Mock
 	private WsContext clientCtx;
+	@Mock
+	private WsContext clientCtx2;
```

Comment:
Could we rename this? I think it's only used in the context of unsubscribing, so maybe something related to that?